Formula/plenv.rb

```diff
@@ -3,8 +3,7 @@ class Plenv < Formula
   homepage "https://github.com/tokuhirom/plenv"
   url "https://github.com/tokuhirom/plenv/archive/2.3.1.tar.gz"
   sha256 "12004cfed7ed083911dbda3228a9fb9ce6e40e259b34e791d970c4f335935fa3"
-  # license ["Artistic-1.0", "GPL-1.0"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "Artistic-1.0"
+  license any_of: ["Artistic-1.0", "GPL-1.0"]
```

Comment:
```suggestion
  license any_of: ["Artistic-1.0-Perl", "GPL-1.0-or-later"]
```

Matches the license of [`perl`](https://github.com/Homebrew/homebrew-core/blob/b6af2713c9e0381fc21dcac3d97207287f3362b8/Formula/perl.rb#L6)